dll/cpl/sysdm/lang/bg-BG.rc

```diff
@@ -278,17 +278,38 @@ BEGIN
     PUSHBUTTON "Отказ", IDCANCEL, 195, 224, 50, 14
 END
 
-IDD_EDIT_VARIABLE DIALOGEX 10, 15, 227, 71
-STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
+IDD_EDIT_VARIABLE DIALOGEX 10, 15, 372, 71
+STYLE DS_CENTER | DS_MODALFRAME | DS_SHELLFONT | WS_POPUPWINDOW | WS_CAPTION | WS_VISIBLE | WS_THICKFRAME
 CAPTION "Обработка на променливата"
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
-    LTEXT "&Име на променливата", IDC_STATIC, 7, 14, 68, 8
-    EDITTEXT IDC_VARIABLE_NAME, 88, 12, 135, 12, ES_AUTOHSCROLL
-    LTEXT "&Стойност на променливата", IDC_STATIC, 7, 32, 68, 8
-    EDITTEXT IDC_VARIABLE_VALUE, 88, 30, 135, 12, ES_AUTOHSCROLL
-    DEFPUSHBUTTON "Добре", IDOK, 116, 50, 50, 14
-    PUSHBUTTON "Отказ", IDCANCEL, 170, 50, 50, 14
+    LTEXT "&Име на променливата:", IDC_STATIC, 7, 14, 50, 8
+    EDITTEXT IDC_VARIABLE_NAME, 75, 12, 289, 12, ES_AUTOHSCROLL
+    LTEXT "&Стойност на променливата:", IDC_STATIC, 7, 32, 50, 8
+    EDITTEXT IDC_VARIABLE_VALUE, 75, 30, 289, 12, ES_AUTOHSCROLL
+    DEFPUSHBUTTON "Добре", IDOK, 260, 50, 50, 14
+    PUSHBUTTON "Отказ", IDCANCEL, 314, 50, 50, 14
+    PUSHBUTTON "Browse Directory...", IDC_BUTTON_BROWSE_FOLDER, 6, 50, 75, 14
+    PUSHBUTTON "Browse Files...", IDC_BUTTON_BROWSE_FILE, 86, 50, 75, 14
+END
+
+IDD_EDIT_VARIABLE_FANCY DIALOGEX 10, 15, 300, 250
+STYLE DS_CENTER | DS_MODALFRAME | DS_SHELLFONT | WS_POPUPWINDOW | WS_CAPTION | WS_VISIBLE | WS_THICKFRAME
+CAPTION "Обработка на променливата"
+FONT 9, "MS Shell Dlg", 0, 0, 0x0
```

Comment:
```suggestion
FONT 8, "MS Shell Dlg", 0, 0, 0x0
```

Reply:
@HBelusca Oh no, don't do that.
I did this in first place since Windows 10 also does this. (Otherwise the UI will look weird with smaller font)

Reply:
Please use the **standard** font setting used for all dialogs in the given resource files. (See all those in these files and elsewhere in all the other GUI apps in ReactOS). Only the CJK languages use font 9.

Reply:
@HBelusca I already knew about standardisation but I thought that an exemption could have been made for this.
Oh well then, going to change it to required size.

Reply:
OK it actually looks fine.
![image](https://user-images.githubusercontent.com/35190743/135761441-7819ecb7-c1ae-48ac-ad02-b6cc51f7c391.png)